src/views/Home.js

```diff
@@ -28,33 +36,56 @@ class Home extends React.Component {
 
   componentDidMount() {
     Home.fetchData(this.props.dispatch);
+    window.addEventListener('resize', this.handleResize);
   }
 
+  componentWillUnmount() {
+    window.removeEventListener('resize', this.handleResize);
+  }
+
+  handleResize() {
+    this.setState({ isMobile: window.innerWidth < 768 });
+  }
+
+
   render() {
     const {formatMessage} = this.props.intl;
     const {topHearing, openHearings, language} = this.props;
-    return (<div className="container">
-      <Helmet title={formatMessage({id: 'welcome'})}/>
-      <h1><FormattedMessage id="welcome"/></h1>
-      <p className="lead"><FormattedMessage id="welcomeMessage"/></p>
-      {topHearing && <FullWidthHearing hearing={topHearing}/>}
-      <hr />
-      <Row>
-        {openHearings && !openHearings.isFetching &&
+    const {isMobile} = this.state;
+    const hearingMap = (openHearings && openHearings.data ? (<div className="map">
+      <h4><FormattedMessage id="open-hearings-on-map"/></h4>
+      <OverviewMap hearings={openHearings.data} style={{width: '100%', height: isMobile ? '70%' : 600}} />
+    </div>) : null);
+
+    return (<div>
+      <div className="container">
+        <Helmet title={formatMessage({id: 'welcome'})}/>
+        <h1><FormattedMessage id="welcome"/></h1>
+        <p className="lead"><FormattedMessage id="welcomeMessage"/></p>
+        {topHearing && <FullWidthHearing hearing={topHearing}/>}
+        <hr />
+        <Row>
+          {openHearings && !openHearings.isFetching &&
+            <Col xs={12}>
+              <div className="list">
+                <h2 className="page-title"><FormattedMessage id="openHearings"/></h2>
+                <HearingCardList hearings={orderBy(openHearings.data, ['close_at'], ['desc'])} language={language}/>
+                <Link className="fullwidth" to="/hearings/list"><FormattedMessage id="allHearings"/></Link>
+              </div>
+            </Col>
+          }
           <Col xs={12}>
-            <div className="list">
-              <h2 className="page-title"><FormattedMessage id="openHearings"/></h2>
-              <HearingCardList hearings={orderBy(openHearings.data, ['close_at'], ['desc'])} language={language}/>
-              <Link className="fullwidth" to="/hearings"><FormattedMessage id="allHearings"/></Link>
+            <div className="feedback-box">
+              <a href="mailto:dev@hel.fi?subject=Kerro kantasi -palaute">
+                <h2 className="feedback-prompt"><FormattedMessage id="feedbackPrompt"/></h2>
+              </a>
             </div>
           </Col>
-        }
-        <Col xs={12}>
-          <div className="feedback-box">
-            <a href="mailto:dev@hel.fi?subject=Kerro kantasi -palaute">
-              <h2 className="feedback-prompt"><FormattedMessage id="feedbackPrompt"/></h2>
-            </a>
-          </div>
+        </Row>
+      </div>
+      <Row>
+        <Col sm={12}>
+          {hearingMap}
```

Comment:
The map doesn't display any geometry; we should fetch the open hearings on the front page with `?include=geojson` to get the geometry on the map.